python-api/gstswitch/helpers.py

```diff
@@ -87,7 +90,7 @@ def audio_port(self, audio_port):
     @property
     def running_tests_video(self):
         """Get the currently running test video list"""
-        return self._running_tests_video
+        return list(self._running_tests_video)
```

Comment:
The problem here is that it's a property returning a copy, which is very weird behaviour that will most definitely bite people unexpectedly. 

If `running_tests_video` was just a list on an object the following things would work;
```python
a.running_tests_video.append('a')
```

Another example would be;
```python
a1 = t.running_tests_video
a2 = t.running_tests_video
assert a1 is a2
``` 

Either we should make it a function (then returning a list is fine) or we should make it a tuple (so that people don't accidently modify it). Happy with either solution.